sdk/core/azure-servicemanagement-legacy/tests/recordings/test_legacy_mgmt_misc.test_add_data_disk_from_disk_name.yaml

```diff
@@ -223,7 +223,7 @@ interactions:
         \ Foundation Server 2013 Update 4 on Windows Server 2012 R2</Label><Location>East\
         \ Asia;Southeast Asia;Australia East;Australia Southeast;Brazil South;North\
         \ Europe;West Europe;Japan East;Japan West;Central US;East US;East US 2;North\
-        \ Central US;South Central US;West US</Location><LogicalSizeInGB>128</LogicalSizeInGB><Name>03f55de797f546a1b29d1b8d66be687a__Team-Foundation-Server-2013-Update4-WS2012R2</Name><OS>Windows</OS><Eula>http://www.microsoft.com/en-us/download/details.aspx?id=13350</Eula><Description>Microsoft\
+        \ Central US;South Central US;West US</Location><LogicalSizeInGB>128</LogicalSizeInGB><Name>03f55de797f546a1b29d1b8d66be687a__Team-Foundation-Server-2013-Update4-WS2012R2</Name><OS>Windows</OS><Eula>http://www.microsoft.com/download/details.aspx?id=13350</Eula><Description>Microsoft\
```

Comment:
Don't update YAML files, those are recordings of actual call for the service

Reply:
Thanks i'll make these changes in this PR - #12697